frontend/src/modules/userManagement/views/addUserView.js

```diff
@@ -96,6 +103,21 @@ define(function(require){
         title: "Couldn't add user",
         text: data.responseText || error
       });
+    },
+
+    togglePasswordView: function() {
+      event && event.preventDefault();
+      
+      var passwordField = document.querySelector('.reveal-input-password');
```

Comment:
In adapt this type of selector isn't common, we tend to use jQuery selectors, please could you update this

```
var passwordField = this.$('.reveal-input-password');
var passwordReveal = this.$('.reveal-password-button');
```

You'll have to use attr like so

`passwordField.attr('type', 'text');`